scripts/ninja.js

```diff
@@ -1032,6 +1032,7 @@ ${ninjaQuickBuidList([
     ocamlDepForBscAsync(othersFiles, othersDir, depsMap),
   ]);
   var jsOutput = generateNinja(jsDepsMap, jsTargets, ninjaCwd, externalDeps);
+  // TODO: ensure that js_string(2).cmi appears before js_json.cm(j|i)
```

Comment:
Running `./scripts/ninja.js clean`, `./scripts/ninja.js config` will result jscomp/others/build.ninja being generated incorrectly.  I got around this by hand editing the generated file of course the hand edits disappear the next time `ninja.js config` is run so a more robust solution is required.